dhis-2/dhis-services/dhis-service-schema/src/test/java/org/hisp/dhis/schema/TranslatablePropertyIntrospectorTest.java

```diff
@@ -37,16 +37,24 @@
 import org.hisp.dhis.DhisSpringTest;
 import org.hisp.dhis.dataelement.DataElement;
 import org.hisp.dhis.dataset.DataSet;
+import org.hisp.dhis.predictor.Predictor;
 import org.hisp.dhis.schema.introspection.TranslatablePropertyIntrospector;
 import org.junit.Test;
 
 public class TranslatablePropertyIntrospectorTest extends DhisSpringTest
 {
+    private TranslatablePropertyIntrospector introspector;
+
+    @Override
+    protected void setUpTest()
+        throws Exception
+    {
+        introspector = new TranslatablePropertyIntrospector();
```

Comment:
You can just assign it directly to the field and make it `final` - no need to delay this to setup.